docs/getting_started/integration_tests.md

```diff
@@ -112,6 +112,8 @@ func (s *ApiTestSuite) SetupApiDefinitions() apiserver.Definer {
 }
 ```
 
+_SetupApiDefinitions_ is needed if you want to run _resty.Client_ or  _ApiServerTestCase_ test cases, more details [here](../advanced_topics/integration_tests.md).  Not adding it will cause an error: `the suite has to implement the TestingSuiteApiDefinitionsAware interface`.
```

Comment:
But... the next tutorial doesn't actually provide more information about this method, does it? And why can't we explain it directly here?